components/MetroBarChart.vue

```diff
@@ -19,31 +19,38 @@
 {
   "ja": {
     "{range}の利用者数*の平均値を基準としたときの相対値": "{range}の利用者数*の平均値を基準としたときの相対値",
-    "都営地下鉄4路線の自動改札出場数": "都営地下鉄4路線の自動改札出場数"
+    "都営地下鉄4路線の自動改札出場数": "都営地下鉄4路線の自動改札出場数",
+    "1月20日~1月24日": ""
```

Comment:
一応日本語も登録しておいた方がいいかと（登録漏れと区別付けるために）
```suggestion
    "1月20日~1月24日": "1月20日~1月24日"
```